docs/endpoints/files/browse.json.md

```diff
@@ -20,7 +20,7 @@ Query parameters:
 
 
 ### AUTHENTICATION ###
-This endpoint requires authentication iff HTTP authentication is
+This endpoint requires authentication if HTTP authentication is
```

Comment:
We should leave as 'iff' since this is a short form of "if and only if". Ditto on the other cases.

Reply:
Since it is documentation, would it be better not to use acronym ? So that translation of the documentation maybe access by machine or the reader easier.

Reply:
Sure, however this file is actually generated from source, so the change would need to be made there:
https://github.com/apache/mesos/blob/1.4.0/src/files/files.cpp#L352-L371
https://github.com/apache/mesos/blob/1.4.0/3rdparty/libprocess/include/process/help.hpp#L82-L90

I think @bbannier was looking into whether we could remove generated markdown since they're no longer needed. We now generate them at the time of publishing the website.

Reply:
I created to track removal of these files; the fix is in principal trivial, we just need to make sure to not break existing tooling.

Regarding this change, I was educated by a native speaker that the spelling here is actually valid and intended when I tried to perform a similar change, https://reviews.apache.org/r/46934/.

Reply:
Noted, I wasn't familiar with this short form. I'll remove these edits.